CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(main)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIERD ON)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
+
+set(OpenCV_DIR /home/wasx/opencv-3.4.9/build)
```

Comment:
you do not need to set the dir if you only have one copy of opencv_lib on your ubuntu.